### PR TITLE
Provide a human readable format when inspecting an RSpec::Core::Example

### PR DESCRIPTION
**Pull request by gavingmiller**
To address issue #1922

### CODE REVIEW

File: lib/rspec/core/example.rb [hunk 1]
```diff
@@ -170,6 +170,12 @@ def initialize(example_group_class, description, user_metadata, example_block=ni
         @reporter = RSpec::Core::NullReporter
       end
 
+      # Provide a human-readable representation of this class
+      def inspect
+        "#<#{self.class.name} #{description.inspect}>"
+      end
+      alias to_s inspect
```
Comment (myronmarston) [inline lib/rspec/core/example.rb]: Is there a spec for the `to_s` alias?  I didn't see one.

**Comment by myronmarston** [comment]
Thanks, @gavingmiller!  Looking good except for the lack of test coverage for the `to_s` alias.  It'd also be nice to have this all squashed into one commit.

File: lib/rspec/core/example.rb (hunk 1 above)
Comment (gavingmiller) [inline lib/rspec/core/example.rb]: There is now! :wink:

**Comment by gavingmiller** [comment]
@myronmarston changes made - let me know if there's anything else.

**Comment by myronmarston** [comment]
Thanks @gavingmiller!

**Pull request merged**